siliconcompiler/core.py

```diff
@@ -953,7 +953,7 @@ def add(self, *args, cfg=None, field='value'):
             if not isinstance(key,str):
                 self.logger.error(f"Key [{key}] is not a string [{args}]")
```

Comment:
I think we should delete the old checks now that we have it inside `_search()`. Same applies to set().

Reply:
Good idea, done.